docs/posix-secondaries.adoc

```diff
@@ -0,0 +1,86 @@
+
+Configuration params that are worth mentioning are located in [uptane] section:
+
+* `port` - TCP port to listen for a connection from Primary
+* `primary_ip` - IP address of Primary ECU
+* `primary_port` - TCP port that Primary's aktualizr listen on for a connection from Secondary
+
+More details on the configuration in general and specific parameters can be found here link:../docs/configuration.adoc[configuration details]
+
+*Run*
+
+``<build-dir>/src/aktualizr_secondary/aktualizr-secondary -c <src-root>/config/posix_secondary.toml``
+Once it's started, you should see a message saying that aktualizr-secondary is listening on the port specified in the config.
+
+
+== *Primary*
+
+*Build*
+
+The Primary executable is built by default, thus following build instructions outlined in link:../README.adoc[the repo's README] should do the trick. If the build procedure is completed successfully then aktualizr executable can be found in ``<build-dir>/src/aktualizr_primary/aktualizr``.
+
+*Configure*
+
+A default configuration of aktualizr acting as Primary can be found in link:../config/sota_local_with_secondaries.toml[<src-root>/config/sota_local_with_secondaries.toml].
+
+Configuration param that is worth mentioning is `[uptane]: secondary_config_file` which specifies a path to secondary(ies) configuration file containing input parameters for aktualizr/Primary to communicate with the secondary(ies). This is a link:../config/posix_secondary_config.json[default secondary configuration for Primary]. In order to use the given default config file please, either copy it into your current working directory or update `[uptane]: secondary_config_file` value so it defines a full path to the secondary config file (e.g. ``<src-root>/config/posix_secondary_config.json``).
+
+Configuration params of secondaries for Primary/aktualizr that are worth mentioning are:
+
+* `secondaries_wait_port` - TCP port aktualizr listen on for connections from Secondaries
+* `secondaries_wait_timeout` - timeout (in sec) of waiting for connections from Secondaries. Primary/aktualizr waits for a connection from those secondaries that it failed to connect to at the startup time.
+* `secondaries` -  a list of Secondary TCP/IP addresses
+
+Please, put your credential.zip file into the current working directory or update `[provision] provision_path` in link:../config/sota_local_with_secondaries.toml[the config] so it specifies a full path to your credential file.
+
+*Run*
+
+``<build-dir>/src/aktualizr_primary/aktualizr -c <src-root>/config/sota_local_with_secondaries.toml``
+Once it's running, you should see a lot of messages, *_'Adding Secondary to Aktualizr'_* and *_'Provisioned successfully on Device Gateway'_* are the key one and are signs of successful start. As result, you should see that a new device has been registered at the backend and it(the device) includes two ECUs, you local/"fake" Primary and Secondary, so you can play with it now...
```

Comment:
I might rephrase this just a bit:
Once aktualizr is running, check the output for *_'Adding Secondary to Aktualizr'_* and *_'Provisioned successfully on Device Gateway'_*. You should then see that a new device has been registered on the server and that it includes two ECUs: your local Primary and your fake Secondary. You can now send updates to either ECU.

Reply:
NP, I would add at the end - "and hope it will work..." :)